### PR TITLE
[IoT Central] Add new sku name to iotcentral

### PR DESCRIPTION
**Pull request by poanchen**
* Adding support for the new sku name (ST0, ST1, ST2) for IoT Central app.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli-testsdk/azure/cli/testsdk/preparers.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def live_only_execute(self, cli_ctx, command, expect_failure=False):
 class ResourceGroupPreparer(NoTrafficRecordingPreparer, SingleValueReplacer):
     def __init__(self, name_prefix='clitest.rg',
                  parameter_name='resource_group',
-                 parameter_name_for_location='resource_group_location', location='westus',
+                 parameter_name_for_location='resource_group_location', location='eastus',
```
Comment (r-delgadillo) [inline src/azure-cli-testsdk/azure/cli/testsdk/preparers.py]: nit: Should we set this to `unitedstates`

File: src/azure-cli/setup.py [hunk 2]
```diff
@@ -90,7 +90,7 @@
     'azure-mgmt-eventhub~=2.6',
     'azure-mgmt-hdinsight~=1.3.0',
     'azure-mgmt-imagebuilder~=0.2.1',
-    'azure-mgmt-iotcentral~=1.0',
+    'azure-mgmt-iotcentral~=2.0.0',
```
Comment (r-delgadillo) [inline src/azure-cli/setup.py]: Is this the correct version? The publishing date looks correct but the features section doesn't seem to have your changes.
image

Comment (r-delgadillo) [reply src/azure-cli/setup.py]: Here is the link to the python SDK. https://pypi.org/project/azure-mgmt-iotcentral/

Comment (kawayiboy) [reply src/azure-cli/setup.py]: build fail is also related to this
pip check
azure-cli 2.0.79 has requirement azure-mgmt-iotcentral~=2.0.0, but you have azure-mgmt-iotcentral 1.0.0.

Comment (poanchen) [reply src/azure-cli/setup.py]: @r-delgadillo This is indeed the correct version, as I am able to create/update to ST1/ST2 in az dev environment with the version 2.0.0
@kawayiboy tu, already made new commits to fix it.

File: src/azure-cli-testsdk/azure/cli/testsdk/preparers.py (hunk 1 above)
Comment (haroldrandom) [reply src/azure-cli-testsdk/azure/cli/testsdk/preparers.py]: Why change this?

Comment (poanchen) [reply src/azure-cli-testsdk/azure/cli/testsdk/preparers.py]: the location 'westus' no longer works in iotc, we need to change it to something else, for example, 'eastus'
As you suggested, I passed location='eastus' to the decorator.

**Comment by poanchen** [comment]
Looks like the tests are failing because I used the Dogfood as environment when generating record session yml and not the prod environment.

`2 - (<Request (GET) https://api-dogfood.resources.windows-int.net/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2019-07-01>).
Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher']
Matchers failed :
host - assertion failure :
management.azure.com != api-dogfood.resources.windows-int.net`

Is there any way we can bypass that? @haroldrandom

**Comment by haroldrandom** [comment]
> Looks like the tests are failing because I used the Dogfood as environment when generating record session yml and not the prod environment.
> 
> `2 - (<Request (GET) https://api-dogfood.resources.windows-int.net/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2019-07-01>). Matchers succeeded : ['method', 'scheme', 'port', 'path', '_custom_request_query_matcher'] Matchers failed : host - assertion failure : management.azure.com != api-dogfood.resources.windows-int.net`
> 
> Is there any way we can bypass that? @haroldrandom

In my local run of your test, it's quite straight forward, an assertion failed. Please update your test accordingly.
```
====================================================================================================================================== FAILURES ======================================================================================================================================
_________________________________________________________________________________________________________________________ IoTCentralTest.test_iotcentral_app _________________________________________________________________________________________________________________________ 
[gw0] win32 -- Python 3.6.8 C:\Users\jianzen\Documents\repo\venv36\Scripts\python.exe

self = <azure.cli.command_modules.iotcentral.tests.latest.test_iotcentral_commands.IoTCentralTest testMethod=test_iotcentral_app>, resource_group = 'clitest.rgnzzzva56ok5mso43plnq5oo5fuos4a2tgvz4l7kwa43tz7nblzs52bln5ojhpw36k', resource_group_location = 'eastus'

    @ResourceGroupPreparer(location='eastus')  # name_prefix not required, but can be useful
    def test_iotcentral_app(self, resource_group, resource_group_location):
        app_name = self.create_random_name(prefix='iotc-cli-test', length=24)
        template_app_name = app_name + '-template'
        template_app_display_name = "My Custom App Display Name"
        rg = resource_group
        location = resource_group_location
        template = 'iotc-default@1.0.0'
        updatedName = app_name + 'update'
    
        # Test 'az iotcentral app create'
        self.cmd('iotcentral app create -n {0} -g {1} --subdomain {2} --sku {3}'.format(app_name, rg, app_name, 'ST2'), checks=[
            self.check('resourceGroup', rg),
            self.check('location', location),
            self.check('subdomain', app_name),
            self.check('displayName', app_name),
>           self.check('sku.name', 'ST2')])
```

**Comment by poanchen** [comment]
I made some changes, all the tests are now passing. Please review and merge. Thank you! @haroldrandom

**Comment by haroldrandom** [comment]
@MyronFanQiu Please double check.

**Pull request merged**